docs/configuration/features/date_features.md

```diff
@@ -18,35 +11,45 @@ preprocessing:
   datetime_format: "%d %b %Y"
 ```
 
-## Date Input Features and Encoders
+Ludwig will try to infer the date format automatically, but a specific format can be provided. The date string spec is
+the same as the one described in python's [datetime](https://docs.python.org/3/library/datetime.html#strftime-strptime-behavior).
 
-Input date features are transformed into a int valued tensors of size `N x 8` (where `N` is the size of the dataset and the 8 dimensions contain year, month, day, weekday, yearday, hour, minute and second) and added to HDF5 with a key that reflects the name of column in the dataset.
+Here are some preprocessing parameters.
 
-Currently there are two encoders supported for dates: Embed Encoder and Wave encoder which can be set by setting `encoder` parameter to `embed` or `wave` in the input feature dictionary in the configuration (`embed` is the default one).
+- `missing_value_strategy` (default `fill_with_const`): what strategy to follow when there's a missing value. The value should be one of `fill_with_const` (replaces the missing value with a specific value specified with the `fill_value` parameter). The special value `backfill` replaces the missing values with the next valid value.
+- `fill_value` (default `""`): the value to replace missing values with when `missing_value_strategy` is `fill_value`. This can be a datetime string. If empty, the current datetime will be used.
```

Comment:
```suggestion
- `fill_value` (default `""`): the value to replace missing values with when `missing_value_strategy` is `fill_with_const`. This can be a datetime string. If empty, the current datetime will be used.
```